### PR TITLE
Add create mail controller

### PR DESCRIPTION
**Pull request by rnnrn**
The only commit that is showing up in the diff is [add controller for create mail](https://github.com/Recursively/shiny-meme/commit/97262b140bb3ae2b7ef78056f3a1de98d3bf4116), cos the rest should already b in master 

@Recursively @hc09141 wanna sanity check?

### CODE REVIEW

File: app/controllers/create_mail_controller.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    @create_mail_event = CreateMail.find(params[:id])
+  end
+
+  def new
+    @create_mail_event = CreateMail.new
+  end
+
+  def create
+    @create_mail_event = CreateMail.new(
+      weight: params[:weight].to_i,
```
Comment (Recursively) [inline app/controllers/create_mail_controller.rb]: Take a look at rails strong params for this

Comment (Recursively) [inline app/controllers/create_mail_controller.rb]: @rnnrn hola

Comment (rnnrn) [inline app/controllers/create_mail_controller.rb]: ya - will do later 2nite

**Comment by rnnrn** [comment]
@Recursively brah how this?

File: app/assets/javascripts/create_mail.js [hunk 2]
```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
```
Comment (Recursively) [inline app/assets/javascripts/create_mail.js]: plz delete this files if they're empty

File: app/assets/stylesheets/create_mail.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the CreateMail controller here.
```
Comment (Recursively) [inline app/assets/stylesheets/create_mail.scss]: this too

File: app/controllers/create_mail_controller.rb [hunk 4]
```diff
@@ -0,0 +1,24 @@
+class CreateMailController < ApplicationController
+
```
Comment (Recursively) [inline app/controllers/create_mail_controller.rb]: no newline needed here

File: app/controllers/create_mail_controller.rb [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+  def create
+    @create_mail_event = CreateMail.create(mail_params)
+  end
+
+  private
+  def mail_params
+    params.permit(:weigth, :volume, :priority, :origin, :destination, :revneue, :expenditure)
+  end
+
```
Comment (Recursively) [inline app/controllers/create_mail_controller.rb]: same here

File: app/helpers/create_mail_helper.rb [hunk 6]
```diff
@@ -0,0 +1,2 @@
+module CreateMailHelper
```
Comment (Recursively) [inline app/helpers/create_mail_helper.rb]: unused plz delete

File: spec/helpers/create_mail_helper_spec.rb [hunk 7]
```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
```
Comment (Recursively) [inline spec/helpers/create_mail_helper_spec.rb]: delete this file if you delete the helper

File: app/controllers/create_mail_controller.rb [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+  def new
+    @create_mail_event = CreateMail.new
+  end
+
+  def create
+    @create_mail_event = CreateMail.create(mail_params)
+  end
+
+  private
```
Comment (Recursively) [inline app/controllers/create_mail_controller.rb]: I personally like a space between the private declaration and all subsequent methods

**Comment by Recursively** [comment]
@rnnrn comments on pr

File: app/controllers/create_mail_controller.rb (hunk 5 above)
Comment (rnnrn) [inline app/controllers/create_mail_controller.rb]: i see u didnt notice the revenue typo but ok ill fix the whitespace

**Pull request merged**